clusterloader2/pkg/test/test.go

```diff
@@ -27,11 +27,11 @@ import (
 var (
 	// CreateContext global function for creating context.
 	// This function should be set by Context implementation.
-	CreateContext CreatContextFunc
+	CreateContext = createSimpleContext
```

Comment:
If you're anyway hard-coding this value, is there a point in offering such extension?

Reply:
This is a default value. If other implementation is needed, It can always be replaced in runtime (e.g. using `init()` function from other implementation package).

Reply:
Ok, that sounds reasonable to me.